### ISSUE TITLE
Support for wildcard in URL field

### ISSUE THREAD

**Issue by krazykipa** [pfn/keepasshttp#199]
Hello,

I use KeepassHTTP to manage all the logins to web portals of various devices in our private network. These do not have DNS entries but are accessed simply by IP.

Without going into too much detail, there are many, many IPs with a common network or host portion of the address, which all use the same username and password to login.

For example, imaging logging into a home router:

192.168.1.1

...but now you have over a hundred of these routers to manage. Thankfully they are all addressed such that:

192.168.x.1 with x being a random number between 0 and 255

I would like to be able to catch all these cases with one entry in the Keepass database, such as 192.168.*.1

Using references within Keepass would work, but duplicating entries enough times to cover all devices we manage would be impractical.

**Comment by carestad** [comment]
I would also appreciate this, being able to use it in the same manner. I've tried setting the URL field to just https://192.168 for instance but that doesn't work either. Would love to be able to set it to just https://192.168.% or https://192.168.*, or even better https://192.168.*.1 as mentioned above.

**Comment by rkresha** [comment]
Agree. For me, it's a matter of port... (e.g. http://192.168.0.99:552 would pull its U:P from an entry with title http://192.168.0.99:*).

**Comment by dev88m** [comment]
@pfn  Are you planning to implement this functionality?

**Comment by pfn** [comment]
I only plan to accept PRs that would implement functionality. I do not plan
to add new features.
